c3.js

```diff
@@ -154,6 +173,8 @@
         var $$ = this, d3 = $$.d3, config = $$.config;
         var defs, main, binding = true;
 
+        $$._axis = new _Axis($$);
```

Comment:
Can you remove `_`? `$$.axis` seems better from the uniformity point of view (other internal variables do not have prefix `_`).